run_app.py

```diff
@@ -40,7 +40,7 @@ def write(self, message):
 
```

Comment:
```suggestion
#pylint: disable=cyclic-import
```

Reply:
also, add `#pylint: disable: consider-using-with`  later in the file to supress the error on line 63